patches/net/minecraft/client/renderer/PostChain.java.patch

```diff
@@ -0,0 +1,57 @@
+--- a/net/minecraft/client/renderer/PostChain.java
++++ b/net/minecraft/client/renderer/PostChain.java
+@@ -17,6 +_,8 @@
+ import java.util.stream.Collectors;
+ import java.util.stream.Stream;
+ import javax.annotation.Nullable;
++
++import net.minecraft.client.renderer.PostPass.TargetInput.BufferType;
```

Comment:
Dang. I *did* check this originally but then I renamed the class and IntelliJ "helped" :unamused: